geonode_dominode/geonode_dominode/settings.py

```diff
@@ -159,3 +159,18 @@
 
 # Add your specific LDAP configuration after this comment:
 # https://docs.geonode.org/en/master/advanced/contrib/#configuration
+
+if DEBUG:
+    LOGGING['loggers']['geonode_dominode'] = {
+        "handlers": ["console"], "level": "DEBUG"
+    }
+    LOGGING['handlers']['console']['level'] = 'DEBUG'
+
+
+BROKER_URL = CELERY_BROKER_URL
+CELERY_TASK_QUEUES += (
+    Queue(
+        'geonode_dominode',
+        GEONODE_EXCHANGE,
+        routing_key='geonode_dominode'),
+)
```

Comment:
I don't think we need a custom queue for this, the default one should suffice

Reply:
The default one doesn't work.

Reply:
why?